public/app/plugins/datasource/prometheus/partials/query.editor.html

```diff
@@ -45,6 +45,8 @@
 			<div class="gf-form-select-wrapper width-8">
 				<select class="gf-form-input gf-size-auto" ng-model="ctrl.target.format" ng-options="f.value as f.text for f in ctrl.formats" ng-change="ctrl.refresh()"></select>
 			</div>
+			<gf-form-switch class="gf-form" label="Instant" label-class="width-4" checked="ctrl.target.instant" on-change="ctrl.refresh()">
```

Comment:
One last remark :) I would have fixed it myself but we currently don't merge to master. 

Please make `label-class="width-4"` `label-class="width-5"`

After this the PR is ready for merge.

Reply:
changed :)